src/workspaces/NewWorkspaceModal/NewWorkspaceModal.tsx

```diff
@@ -209,12 +209,15 @@ export const NewWorkspaceModal = withDisplayName(
             true
           );
 
-          // Wait for the default WDS instance to exist.
+          // Wait for the default WDS collection to exist.
           const proxyUrl = wds!.proxyUrls.wds;
           await Utils.poll(
             async () => {
-              const instances: string[] = await Ajax().WorkspaceData.listInstances(proxyUrl);
-              if (instances.includes(createdWorkspace.workspaceId)) {
+              const collections: string[] = await Ajax().WorkspaceData.listCollections(
+                proxyUrl,
+                createdWorkspace.workspaceId
```

Comment:
I'm getting this ticket over the line for Bria (who just went on vacation) so I'm not 100% sure, but I think the check serves to determine whether the call succeeded, or if it needs to be attempted again. 

(I'm looking at the comment a few lines above that reads `// Wait for the default WDS collection to exist.` and the `shouldContinue` property in the returned object.)

@davidangb, do you know for sure?

Reply:
I just updated the test

Reply:
I would like to better understand this API before approving the PR. Maybe it's really a matter of whether or not `collections` is empty?

Reply:
The API in question returns a list of all collections in a given workspace.

_By convention_, the collection shown in the UI is the collection whose id is the same as the workspaceId. There is no guarantee that such a collection exists - though it should, if all WDS startup processes ran successfully.

An analogy for this is the workspace record in Rawls and the workspace resource in Sam. Both have the same id, but the fact that there's a row in the Rawls db does not necessitate that there is a row in the Sam db.

At some future point, when our UX (and documentation, etc) support the ability to display multiple collections in the UI and switch between those collections, it will be ok to check if the returned list of collections is non-empty. For now, we need to check for the convention in which a collection exists with the same id as the workspace.

Reply:
Thanks-- this really helps! @mspector can you add a comment in the code so that future folks don't think, "oh, I can just remove this check because I'm passing the workspace ID to the API"?